Core/src/org/sleuthkit/autopsy/datamodel/Accounts.java

```diff
@@ -77,11 +78,16 @@
 public class Accounts extends Observable implements AutopsyVisitableItem {
```

Comment:
I will try this, I think some of the classes will still need to be public to be seen outside of the accounts package and so there will not be the warnings you suggest.  having a new package will also allow me to move some of the classes into their own files, which I think will make them easier to understand.